libr/bin/format/elf/elf.c

```diff
@@ -2101,8 +2157,26 @@ char* Elf_(r_bin_elf_get_arch)(ELFOBJ *bin) {
 	default: return strdup ("x86");
 	}
 }
+char* Elf_(r_bin_elf_get_features)(ELFOBJ *bin) {
+	Elf_(Ehdr)* ehdr =(Elf_(Ehdr) *) &bin->ehdr;
 
-char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
+	if (ehdr->e_machine == EM_MIPS){
+		if (elfn64_check_arch(ehdr)) {
```

Comment:
Imho those ‘features’ are just abi. Which should be named abi instead of features. And this should define the default calling convention for the analysis. In this pr it is just being used by rbin afaik

Reply:
So should I add a "char *abi" to RbinInfo? But I asked XVilka, and he said it’s okay to put abi information in feature.

Or I change the name of function Elf_(r_bin_elf_get_features)  to Elf_(r_bin_elf_get_abi)

Reply:
@trufae ping

Reply:
maybe it makes sense to have both.. an abi or cc and the whole features string. This way we can check for the default calling convention to use